src/views/components/countryThinFileSummaryList.njk

```diff
@@ -6,7 +6,7 @@
               text: translate("checkDetails.selectedId")
             },
             value: {
-              html: translate("{{ idChoice }}")
+              html: translate("{{ idTranslationKey }}")
```

Comment:
This `translate("{{}})` thing doesn't actually work does it? Shall we just do `{{ idTranslationKey }}`?

Reply:
Yes that's necessary, we get this without it
![Screenshot 2023-11-15 at 16 22 59](https://github.com/govuk-one-login/ipv-cri-f2f-front/assets/117987734/ea55cb3b-1334-4dac-b1cc-eb20e2f2a5c9)